system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiClientTest.java

```diff
@@ -21,15 +21,15 @@
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.eclipse.dataspaceconnector.registration.client.TestUtils.CLIENT_DID_WEB;
 
 @IntegrationTest
 public class RegistrationApiClientTest {
+    static final String CLIENT_DID_WEB = "did:web:did-server:test-client";
     static final String API_URL = "http://localhost:8182/authority";
     static final Faker FAKER = new Faker();
     static RegistryApi api;
 
-    String participantUrl = FAKER.internet().url();
+    String did = "web:did:" + FAKER.internet().url();
```

Comment:
In `client-cli/src/test/java/org/eclipse/dataspaceconnector/registration/cli/TestUtils.java` you use `FAKER.internet().domainName()` instead of `FAKER.internet().url()` as here.

Just double-checking this is on purpose.

Reply:
ah good point,  I'll change it to domainName for consistency